src/main/java/net/dries007/holoInventory/network/BlockInventoryMessage.java

```diff
@@ -53,8 +57,31 @@ public IMessage onMessage(BlockInventoryMessage message, MessageContext ctx)
                     itemStacks[i] = ItemStack.loadItemStackFromNBT(tag);
                     if (itemStacks[i] != null) itemStacks[i].stackSize = tag.getInteger("Count");
                 }
-                if (message.data.hasKey("class")) Renderer.tileMap.put(message.data.getInteger("id"), new NamedData<ItemStack[]>(message.data.getString("name"), message.data.getString("class"), itemStacks));
-                else Renderer.tileMap.put(message.data.getInteger("id"), new NamedData<ItemStack[]>(message.data.getString("name"), itemStacks));
+                NamedData<ItemStack[]> data;
+                if (message.data.hasKey("class"))
+                    data = new NamedData<>(message.data.getString("name"), message.data.getString("class"), itemStacks);
+                else
+                    data = new NamedData<>(message.data.getString("name"), itemStacks);
+                if (Config.enableStacking)
+                {
+                    List<ItemStack> stacks = new ArrayList<>();
```

Comment:
I don't understand how that may work. `stacks` is a local list, so you modify some stacks in the original list, but don't really filter out anythingm because resulting list is ignored

Reply:
.. and that's why we need to have review. Forget to commit the merging back to global store 😅